src/Movies.js

```diff
@@ -4,41 +4,82 @@ import $ from 'jquery';
 import { baseUrl } from './config';
 import urlJoin from 'url-join';
 
+function getAttribute($context, selector, attr) {
+  return $context.find(selector).attr(attr);
+}
+
 function getText($context, selector) {
   return $context.find(selector).text().trim();
 }
-function getAttribute($context, selector, attr) {
-  return $context.find(selector).attr(attr);
+
+function getSpecificContentArray($context, selector, i) {
+  return $($context.find(selector)[i]).text().trim();
+}
+
+function parseCount(str) {
+  if(!str) {
+    return 0;
+  }
+
+  str = str.match(/(\d*,?\d+)\s*titles?/i)[1].split(',');
+
+  if (str[1]){
+    return Number(`${str[0]}${str[1]}`);
+  } else {
+    return Number(str[0]);
+  }
 }
 
 function parseMovie($row) {
-  let title = getText($row, 'td.title > a');
-  let url = getAttribute($row, 'td.title > a', 'href');
-  let rating = parseFloat(getText($row, 'span.rating-rating').split('/')[0]) || undefined;
-  let desc = getText($row, 'span.outline');
-  let imgUrl = getAttribute($row, '.image img', 'src');
-  let genre = getText($row, '.title .genre');
+  let title = getText($row, '.lister-item-header a');
+  let url = getAttribute($row, '.lister-item-header a', 'href');
+  let rating = parseFloat(getText($row, '.ratings-bar strong').split('/')[0]) || undefined;
+  let metascore = parseFloat(getText($row, '.ratings-metascore span').split('/')[0]) || undefined;
+  let runtime = parseFloat(getText($row, '.text-muted .runtime').split('/')[0]) || undefined;
+  let desc = getSpecificContentArray($row, '.text-muted', 2); //Used when there is no specific class on element
+  let imgUrl = getAttribute($row, '.lister-item-image img', 'loadlate');
+  let genre = getText($row, '.text-muted .genre');
+  let votes = getSpecificContentArray($row, '.sort-num_votes-visible span', 1);
+
+  return { title, url, rating, metascore, runtime, desc, imgUrl, genre, votes };
+}
 
-  return { title, url, rating, desc, imgUrl, genre };
+function parseMovieCount(doc) {
+  let $article = $(doc).find('#main .article');
+  let total = parseCount(getText($article, '.lister .nav .desc'));
```

Comment:
```js
let $nav = $article.find('.lister .nav .desc').text().trim();
let total = getTotal($nav.text().trim());

function getTotal(str) {
  let total = str.split(/\s+/)[4]; // ["1", "to", "50", "of", "10,811", "titles", "|", "Next", "»"]
  retrun parseInt(total.replace(/,/g, ''), 10));
}
```
Believe me it is better this way you don't want to hear my comment on `#parseCount`...